### DISCUSSION TITLE
MODS DOESN'T WORK

### DISCUSSION THREAD

**Discussion by Sbst397** [thatskymod/Sky-CotL-Scripts#117] [General]
I'm crying guys, like literally no mods work and Canvas started to not working too, I know I'm not alone so why they don't do anything 😭😭

**Comment by thatskymod** [comment]
Mods in this repository are regularly tested for compatibility. That said, I don't see any apparent issue with the current mods. Be sure to update libLangit to the latest version, [0.4.5](https://github.com/thatskymod/Sky-CotL-Scripts/blob/6a5d6946eb5661e5e14f714d56ca0f4157b3c351/Canvas/Libs/libLangit045.so) and Canvas to [1.5.7](https://github.com/thatskymod/Sky-CotL-Scripts/blob/91728bfbadc25e56bdf01cd8c798b387e2d74014/Canvas/Canvas.apk). Other mods like libsMusic should be removed to prevent Canvas from crashing, but you can keep libRoman to receive the next update once it's ready. Same applies to your other thread, #115.
